text/0000-reactive-assignments.md

```diff
@@ -598,7 +598,20 @@ TODO
 
 ### TypeScript
 
-TODO
+A major advantage of this proposal over Svelte 2 is that components become far more amenable to typechecking and other forms of static analysis. Even without TypeScript, VSCode is able to offer much richer feedback now than with Svelte 2.
+
+An eventual goal is to be able to use TypeScript in components:
+
+```html
+<script type="typescript">
```

Comment:
`<script lang="typescript">` imo – `type` should be reserved for `type="module"` (native).

Reply:
updated